examples/multiple_windows/lib/app/window_controller_render.dart

```diff
@@ -0,0 +1,43 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// ignore_for_file: invalid_use_of_internal_member
+// ignore_for_file: implementation_imports
+
+import 'package:flutter/material.dart';
+import 'regular_window_content.dart';
+import 'models.dart';
+import 'package:flutter/src/widgets/_window.dart';
+
+class WindowControllerRender extends StatelessWidget {
+  const WindowControllerRender({
+    required super.key,
+    required this.controller,
+    required this.onDestroyed,
+    required this.onError,
+    required this.windowSettings,
+    required this.windowManagerModel,
+  });
+
+  final BaseWindowController controller;
+  final VoidCallback onDestroyed;
+  final VoidCallback onError;
+  final WindowSettings windowSettings;
+  final WindowManagerModel windowManagerModel;
+
+  @override
+  Widget build(BuildContext context) {
+    return switch (controller) {
+      final RegularWindowController regular => RegularWindow(
+        key: key,
```

Comment:
This results in a key that's used by both the parent and child. Do we really need to pass the `key` down to `RegularWindow` here? Or is it enough for `WindowControllerRender` to have use the `key`?